components/merkledb/examples/manual_migration.rs

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Shows how to migrate database data. The migration follows the following scenario:
```

Comment:
Nit: I'd move the common part of the migration description to the shared module and refer to it in the examples: "For the description of the migration scenario, see the `migration` module docs." In the example docs, I'd focus on the differences among examples.